go.mod

```diff
@@ -248,6 +248,7 @@ require (
 )
 
 replace (
+	go.etcd.io/etcd/client/v3 => github.com/serathius/etcd/client/v3 v3.0.0-20230829133936-3a794fbb8970
```

Comment:
this is WIP , right?

Reply:
Yes, interface needs to be designed on K8s side and then added to etcd client.

Reply:
I see, maybe you already know, but if you set WIP on the title the bot will mark  it as not mergable